### PR TITLE
Go 1.5 release

### PR DESCRIPTION
**Pull request by antonypp**

### CODE REVIEW

**Comment by calebthompson** [comment]
Should this also update the various tools and other stuff still on 1.4.2?

**Comment by andygrunwald** [comment]
This was released today. See https://blog.golang.org/go1.5
Would love to see this as soon as possible.

**Comment by mistydemeo** [comment]
The code that deals with the bootstrap go needs to be adjusted to work with the stable release as well in this version.

**Comment by mistydemeo** [comment]
@andygrunwald This was only submitted less than ten minutes ago - please be patient, thanks!

**Comment by antonypp** [comment]
@calebthompson change

**Comment by calebthompson** [comment]
We _probably_ don't want the `devel` block anymore as `1.5.0-rc1` has been superseded by the full 1.5 release?

In any case, this looks good.

**Comment by DomT4** [comment]
We also need to look at the situation re [this comment](https://github.com/Homebrew/homebrew/pull/41305#issuecomment-122707705).

**Comment by mistydemeo** [comment]
The `if build.head? || build.devel?` block in the install method needs to be made unconditional.

**Comment by daMupfel** [comment]
:+1:

**Comment by ahmetalpbalkan** [comment]
:+1:

**Comment by mistydemeo** [comment]
We don't need additional +1s, please - thanks!

File: Library/Formula/go.rb [hunk 1]
```diff
@@ -16,72 +16,39 @@ class Go < Formula
   end
 
   devel do
```
Comment (DomT4) [inline Library/Formula/go.rb]: You no longer require this block, at all.

File: Library/Formula/go.rb [hunk 2]
```diff
@@ -16,72 +16,39 @@ class Go < Formula
...
+    (buildpath/"gobootstrap").install resource("gobootstrap")
+    cd "#{buildpath}/gobootstrap/src" do
+      system "./make.bash", "--no-clean"
     end
+    # This should happen after we build the test Go, just in case
+    # the bootstrap toolchain is aware of this variable too.
+    ENV["GOROOT_BOOTSTRAP"] = ENV["GOROOT_FINAL"]
 
     cd "src" do
       targets.each do |os, archs|
```
Comment (DomT4) [inline Library/Formula/go.rb]: You've removed all the logic for `targets` so `targets.each do |os, archs|` is pointing at nothing. You'll need to tweak this line for whatever is necessary now.

Comment (DomT4) [inline Library/Formula/go.rb]: Actually, this whole block from L54-L61 needs changing.

**Pull request closed without merging**